adamsjr8576/palette-picker-fe
# Palettes forms styling

### What does this PR do?
- Adds styling to palette generator and forms
- Adds two circles in each color of the palette that pulls the colors from the left and the right reverses them and displays them within. The circles are tied to the locked functionality.
- Adds the ability to tab through the locked buttons and use enter to toggle.
- Adds dynamic color changes to header and forms based on the random palette generator.
- updates tests to pass based on changes made and adds mapStateToProps testing where needed

### How to test?
- Pull down
- run `npm start`
- You should see the changes reflected - PUSH the create new palette button!

### Issues
- This PR closes no issues

### Notes:
- Let me know what you think about the colors changing on the forms - I kind of like the color pop in there that ties it to the current palette.

<img width="1427" alt="Screen Shot 2020-02-11 at 1 13 07 PM" src="https://user-images.githubusercontent.com/49846853/74275271-cf1e4f00-4cd0-11ea-8e8c-c1797bf364e3.png">
